test/unittests/test_excel2xml.py

```diff
@@ -506,30 +505,6 @@ def test_excel2xml(self) -> None:
         if os.path.isfile("excel2xml-output-data.xml"):
             os.remove("excel2xml-output-data.xml")
 
-    @pytest.mark.filterwarnings("ignore")
```

Comment:
the sample script doesn't exist any more in this sense. It is now everything in the `0123-import-scripts`, which has its own e2e-Test.